package.json

```diff
@@ -2,14 +2,14 @@
   "name": "SparkEd",
   "private": false,
   "scripts": {
-    "start": "meteor",
+    "start": "meteor --allow-incompatible-update",
```

Comment:
I was only able to run it by specifying the version like `meteor --release 2.12`